src/agent/src/config.rs

```diff
@@ -197,6 +197,12 @@ mod tests {
         }
 
         let tests = &[
+            TestData {
```

Comment:
This doesn't seem to be part of this commit??

Reply:
Hi @jodh-intel , it is. 
It's used to test whether the options of "key=value" and "key${appendix}=value" can be parsed
correctly.